chapters/4-acquisition.tex

```diff
@@ -525,137 +521,157 @@ \subsection{Programming electronic questionnaires}
 where you set up your own encryption
 where you are in full control of who has access to the key.
 
+\subsection{Collecting and storing data securely}
+
+Most data collection software will automatically encrypt
+all data in transit (i.e., upload from field or download from server).
+However, it is necessary to ensure that confidential data
+are protected when stored on a server
+owned by the data collection software provider
+or which can be accessed by people not on your research team
+(including your local IT or system administrators).
+In most data collection platforms,
+encryption-at-rest needs to be explicitly enabled and operated by the user.
+When collecting data, the tablets or the browsers used for data collection
+should encrypt all information before submitting it to the server,
+and you should decrypt it only after downloading to your local machine.
+
+This is a perfect use case for \textbf{asymmetric encryption}
+where there are two keys, forming a ``public/private key pair''.
+The public key can safely be sent to all tablets or browsers
+so it can be used for encrypting the data before it is submitted to the server.
+Only the private key in the same key pair can then be used to decrypt that data
+so it can be accessed after it has been received.
+The private key should be kept secret and
+should not be shared with anyone not listed on the IRB.
+Again, you must store the key pair in a secure location,
+such as a secure note in a password manager,
+as there is no way to access your data if the private key is lost.\sidenote{
+  DIME guidelines for encryption in SurveyCTO/ODK are part of the DIME Standards:
+  \url{https://github.com/worldbank/dime-standards/}}
+If your data collection service allows you to browse data in the browser,
+then the encryption is only implemented correctly if you are asked for the key each time.
+
+The data security standards that apply
+when receiving confidential data from the field
+also apply when transferring confidential data to the field,
+such as sampling or enumeration lists containing PII.\index{data transfer}
+In some survey software,
+you can use the same encryption that allows you to receive data securely
+from the field, to also send confidential data,
+such as an identifying list of respondents, to the field.
+Otherwise, you will need to create a securely stored file,
+transfer it to the field team using an insecure tool,
+and have them decrypt the information locally
+using a key that is transferred using a secure password manager.
+This process will be more similar to that for securely \textit{storing} data,
+which we discuss next.
 
-\subsection{Storing data securely}
 % doesn't cover 'big' data
-The first thing you need to do before planning how to securely receive data,
+The first thing you need to do before planning how to securely send or receive data,
 is to plan how to securely store data after you have received it.
```

Comment:
```suggestion
is to plan how to securely store data after it has been transferred.
```
Since we now say "you send or receive" we cannot say "you have received"